_test/test_sqw_class/test_eval.m

```diff
@@ -49,26 +49,35 @@ function test_sqw_eval(obj)
 
             ds = sqw_eval(obj.sqw_obj, @test_eval.sqw_eval_tester, []);
 
-            pix = ds.pix.signal;
-            assertEqual(pix, 2.*ones(size(pix)));
+            sig = ds.pix.signal;
+            assertEqual(sig, 2.*ones(size(sig)));
+            assertEqual(ds.data.s, 2.*ones(size(ds.data.s)));
+            assertEqual(ds.data.e, 2.*zeros(size(ds.data.s)));
         end
 
         function test_sqw_eval_average_fb(obj)
-            clob = set_temporary_config_options(hor_config, 'mem_chunk_size', 1000);
+            clob = set_temporary_config_options(hor_config, 'mem_chunk_size', 8000);
```

Comment:
It would be good if we can comment on changes of size like this

Reply:
As I've told on the meeting, I am going through the tests and optimizing it -- minimizing number of filebacked pages still keeping the dataset filebacked and testing functionality improves test performance